app/src/main/java/org/mozilla/fenix/ext/String.kt

```diff
@@ -21,7 +21,16 @@ fun String.replace(pairs: Map<String, String>): String {
 fun String?.urlToHost(): String {
     return try {
         val url = URL(this)
-        url.host
+        val firstIndex = url.host.indexOfFirst { it == '.' } + 1
+        val lastIndex = url.host.indexOfLast { it == '.' }
```

Comment:
@ekager this is the updated way I'm generating the "host" string because just want the name of the website (e.g. `mozilla` instead of `www.mozilla.org` like we previously had.

Reply:
I cleaned this up quite a bit so it looks nicer and has a comment to explain the logic. Do you think it's okay to have this change here?

Reply:
Is this the only place this extension is used? You may want to create another function just to not break someone else's code using this

Reply:
Good point. I moved it to a separate function :)